tmk_core/protocol/lufa/lufa.c

```diff
@@ -89,6 +89,10 @@ extern keymap_config_t keymap_config;
 #    include "joystick.h"
 #endif
 
+#ifdef EEPROM_DRIVER
+#    include "eeprom_driver.h"
+#endif
+
```

Comment:
Did this part get here by mistake?

Reply:
No, not a mistake. Nobody else ever calls eeprom driver_init(), except chibios main. Which means eeprom driver might not work correctly on non-chibios targets. Arguably this could/should have been a separate commit, but from my perspective it was related to the same work, because I was testing the i2c bitbang code with external i2c eeprom. Would you like me to separate out this change as a separate PR?

Reply:
Probably nobody ever tried to use any of the replacement EEPROM drivers on AVR; I suppose you tried to do that for testing, and it actually worked? In any case, this looks like a completely unrelated change (and in theory it could make sense even without this I2C bitbang driver — e.g., if you wanted to have a larger EEPROM than supported by the MCU).

Reply:
Alright, I removed this change from the most recent PR state. I will add it back as a separate PR.